packages/shared/src/utils/date/date-time.ts

```diff
@@ -274,3 +275,32 @@ export const convertTimeFormat = (time: string) => {
     const time_suffix = `${Number(time_hour) >= 12 ? 'pm' : 'am'}`;
     return `${formatted_time} ${time_suffix}`;
 };
+
+/**
+ *  Get a formatted time since a timestamp.
+ * @param  {Number} timestamp in ms
+ * @return {String} '10s ago' or '1m ago' or '1h ago' or '1d ago' or '1 Jan 2019'
+ */
+export const getTimeSince = (timestamp: number) => {
```

Comment:
An improved version of `timeSince` from Bot. Since it's in shared, it can be reused by Bots too if they need.